market/connector/currencyConverter/README.md

```diff
@@ -0,0 +1,15 @@
+AXON Ivy’s [frankfurter.app](https://www.frankfurter.app) connector helps you to accelerate process automation initiatives by converting currencies from one to another. 
```

Comment:
@andreasbalsiger Plese contribute here; if we need adjustments in the README.md for a first version of this cool connector

Reply:
Axon Ivy’s [Frankfurter currency data API](https://www.frankfurter.app) connector helps you to accelerate process automation initiatives by converting currencies from one to another. The connector:

- features different currency rates based such as latest rates, historical rates, time series and more.
- supports you with an easy-to-copy demo implementation to reduce your integration effort.
- enables low code citizen developers to enhance existing business processes with currency converter features.